packages/nextcloud/lib/src/nextcloud.openapi.dart

```diff
@@ -35,8 +38,8 @@ class NextcloudResponse<T, U> {
   String toString() => 'NextcloudResponse(data: $data, headers: $headers)';
 }
 
-class _Response {
-  _Response(
+class RawResponse {
```

Comment:
If wanted we can still hide it from the export.

Reply:
Yeah, but I don't think it matters that much.